### ISSUE TITLE
Bug: if I modified the configuration of `Prompts.WorkingMemoryName` and `LongTermMemoryName`, you should use them as Key for access `MemoryMap` in `ChatMemoryController`

### ISSUE THREAD

**Issue by AwesomeYuer** [microsoft/chat-copilot#388]
**Describe the bug**
if I modified the configuration of `Prompts.WorkingMemoryName` and `LongTermMemoryName`, you should use them for access in `ChatMemoryController`

**To Reproduce**
1. modified the configuration of `Prompts.WorkingMemoryName` and `LongTermMemoryName`
```json
"Prompts": {
  "WorkingMemoryName": "W",
  "LongTermMemoryName": "L"
}
```
2. access `persona` tab on frontend, will occurs 400 error, that is access the url as below:

`https://localhost:40443/chatMemory/wxuse_2306_b17893d883b34657a462841a8208c938/WorkingMemory`

`https://localhost:40443/chatMemory/wxuse_2306_b17893d883b34657a462841a8208c938/LongTermMemory`

because you didn't use the configuration as above

**Expected behavior**
no error!

**Screenshots**
Fix suggestions

![image](https://github.com/microsoft/chat-copilot/assets/1026479/fd499676-cc66-40ab-ae10-be07c0b8b803)

![image](https://github.com/microsoft/chat-copilot/assets/1026479/38046e7c-2d64-4a3d-93d3-099f757b10a1)

**Platform**
 - OS: [e.g. Windows, Mac]
 - IDE: [e.g. Visual Studio, VS Code]
 - Language: [e.g. C#, Python]
 - Source: [e.g. NuGet package version 0.1.0, pip package version 0.1.0, main branch of repository]

**Additional context**

**Comment by AwesomeYuer** [comment]
found another impact and Fix suggestions:

![image](https://github.com/microsoft/chat-copilot/assets/1026479/c0bc22c4-227b-4bc1-bafe-d5f8e00a2a38)

![image](https://github.com/microsoft/chat-copilot/assets/1026479/9dd089b4-eae7-40b4-98d2-bac09bd622aa)

**Comment by crickman** [comment]
Thank you for reporting this.  I'm familiar with the code/config to which you are referring and and will review.

**Comment by crickman** [comment]
@AwesomeYuer - Curious, do you find it useful that these keys can be configured?  If so, why?

**Comment by AwesomeYuer** [comment]
Sure!
Since I modified your code, changed `chatId` to a `string` from `Guid`, 
and added `UserName` and `HourHourMinuteMinute` as a prefix, it is convenient to find the corresponding Session data by `UserName` in the backend. 
`UserName` may be very long, and I found that `Postgres` has a limit on the length of table names! 
So I modified these config from `LongTermMemory` to `L`  for Trim `ChatId string` length! and Remove `-` in `Guid string`, 
`Qdrant` found no problem for collection name length limited.

Postgres:

![image](https://github.com/microsoft/chat-copilot/assets/1026479/e4c2cb8b-ae14-4875-af9a-0631293471b1)

Qdrant:

![image](https://github.com/microsoft/chat-copilot/assets/1026479/69e19f26-5ae0-4ac8-a13f-8c9d6721d816)
